zap/src/main/java/org/zaproxy/zap/view/popup/PopupMenuItemIncludeInContext.java

```diff
@@ -97,7 +97,7 @@ public void performHistoryReferenceActions(List<HistoryReference> hrefs) {
                         session, ContextIncludePanel.getPanelName(context.getId()), false);
     }
 
-    private void recreateUISharedContexts(Session session) {
+    protected void recreateUISharedContexts(Session session) {
```

Comment:
Revert and introduce the method `normaliseUrl(String)` (or something like that), the other class then just needs to override the new method instead of duplicating `performAction`.